File: src/test/java/test/ChangeConnStateTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class ChangeConnStateTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        EnableExtIdpConnDto request = new EnableExtIdpConnDto();
        request.setAppIds("appIds_4736");
        request.setAppId("appId_6505");
        request.setEnabled(Boolean.TRUE);
        request.setId("id_5707");
        request.setTenantId("tenantId_8119");

        IsSuccessRespDto response = client.changeConnState(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/ChangeConnStateTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.EnableExtIdpConnDto{
        AppIds: "appIds_5057",
        AppId: "appId_3342",
        Enabled: false,
        Id: "id_2862",
        TenantId: "tenantId_5342",
    }
    response := client.ChangeConnState(&request)
    fmt.Println(response)
}

